### PR TITLE
Use NftID instead of UniqueTokenId

### PR DESCRIPTION
**Pull request by kimbor**
**Description**:
1) Move NftID from token_get_nft_info.proto to basic_types.proto
2) Change everywhere that uses UniqueTokenID to use NftID instead
3) Delete UniqueTokenId from common.proto

**Related issue(s)**:
Fixes [#7346 in hedera-services](https://github.com/hashgraph/hedera-services/issues/7346)

**Note**:
DO NOT MERGE UNTIL AFTER [PR # 7409](https://github.com/hashgraph/hedera-services/pull/7409) has been merged

### CODE REVIEW

File: services/state/token/account.proto [hunk 1]
```diff
@@ -97,11 +97,11 @@ message Account {
      */
     int64 head_token_number = 13;
     /**
-     * The NftId of the head of the linked list from unique tokens map for the account.
+     * The NftID of the head of the linked list from unique tokens map for the account.
      */
     int64 head_nft_id = 14;
```
Comment (iwsimon) [inline services/state/token/account.proto:102]: Should this be NftID head_nft_id=14?

Comment (kimbor) [reply services/state/token/account.proto:102]: That change will be made as part of issue #7243

Comment (iwsimon) [reply services/state/token/account.proto:102]: I see. Thanks.

File: services/basic_types.proto [hunk 2]
```diff
@@ -113,6 +113,21 @@ message AccountID {
...
 }
 
+/**
+ * Represents an NFT on the Ledger
+ */
+message NftID {
+    /**
+     * The (non-fungible) token of which this NFT is an instance
+     */
+    TokenID tokenID = 1;
```
Comment (Neeharika-Sompalli) [inline services/basic_types.proto:123]: should be camel case token_id

File: services/basic_types.proto [hunk 3]
```diff
@@ -113,6 +113,21 @@ message AccountID {
...
+
+    /**
+     * The unique identifier of this instance
+     */
+    int64 serialNumber = 2;
```
Comment (Neeharika-Sompalli) [inline services/basic_types.proto:128]: same here serial_number

File: services/basic_types.proto [hunk 4]
```diff
@@ -113,6 +113,21 @@ message AccountID {
     }
 }
 
+/**
+ * Represents an NFT on the Ledger
+ */
```
Comment (Neeharika-Sompalli) [inline services/basic_types.proto:118]: Docs can be taken from the removed `UniqueTokenId` . They seem to be detailed

File: services/basic_types.proto (hunk 2 above)
Comment (kimbor) [reply services/basic_types.proto:123]: Almost all of the names in basic_types.proto are in camel case, but a few are snake case. Do we want to standardize on snake case? If so, there are a lot more places that should be changed.

Comment (Neeharika-Sompalli) [reply services/basic_types.proto:123]: yeah, I think old ones are in camel case. But then we standardized to snake case for any further changes. It should not affect generated code afaik.

Comment (kimbor) [reply services/basic_types.proto]: I fixed it here. I also created issue #7423 to fix it other places.

File: services/basic_types.proto (hunk 3 above)
Comment (kimbor) [reply services/basic_types.proto]: Fixed

File: services/basic_types.proto (hunk 4 above)
Comment (kimbor) [reply services/basic_types.proto]: Fixed

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**